eng/helix/content/RunTests/TestRunner.cs

```diff
@@ -65,6 +66,20 @@ public bool SetupEnvironment()
                 return false;
             }
         }
+        
+        public void DisplayEnvironment()
+        {
+            Console.WriteLine("KeyValue pairs from Environment.GetEnvironmentVariables():");
+            foreach (DictionaryEntry pair in Environment.GetEnvironmentVariables())
+            {
+                Console.WriteLine(pair.Key + "=" + pair.Value);
+            }
+            Console.WriteLine("KeyValue pairs from TestRunner.EnvironmentVariables");
+            foreach (KeyValuePair<string, string> pair in EnvironmentVariables)
+            {
+                Console.WriteLine(pair.Key + "=" + pair.Value);
+            }
+        }
```

Comment:
We do not want to do this because displaying the environment can expose secrets. We're not currently doing Helix runs internally but this is still something we shouldn't do.